README.md

```diff
@@ -20,11 +20,30 @@ This module provides Stackdriver Debugger support for Node.js applications. [Sta
 # Install with `npm` or add to your `package.json`.
 npm install --save @google/cloud-debug
 
-# Require the agent at the top of your main script (but after '@google/cloud-trace' if you are also using it).
-require('@google/cloud-debug');
+# Require and start the agent at the top of your main script (but after '@google/cloud-trace' if you are also using it).
+require('@google/cloud-debug').start();
 ```
 Deploy your application, and navigate to the [Stackdriver Debug view][debug-tab] within the [Google Cloud Console][dev-console] to set snapshots and start debugging.
 
+## Configuration
+
+First, some configuration options can be specified through environment variables:
+
+* `GCLOUD_DEBUG_DISABLE` specifies whether to enable or disable the agent
+* `GCLOUD_DEBUG_LOGLEVEL` specifies the debug level
+* `GCLOUD_DEBUG_REPO_APP_PATH` specifies the path within your repository to the directory containing the `package.json` for your deployed application
+* `GAE_MODULE_NAME` specifies the service name of the deployed application
+* `GAE_MODULE_VERSION` specifies the version of the deployed application
```

Comment:
I would prefer to keep the environment variables undocumented for now specially given that we are planning to reverse the precedence in the future (in a semver-major update). It would be best if fewer people were dependent on these and their semantics.